### ISSUE TITLE
Setting VUE_APP_URL_BASE_API doesn't do anything

### ISSUE THREAD

**Issue by FlomoN** [n8n-io/n8n#1219]
**Describe the bug**
When setting the Environment Variable `VUE_APP_URL_BASE_API` rest calls still go to the normal baseUrl that the frontend also loads with. 

In my example: I want to open the frontend on n8n.example.com behind a traefik ingress, but i want all the api calls (especially SSE, since traefik messes those up) to directly go to the container, bypassing the ingress. I assumed that this would happen when setting `VUE_APP_URL_BASE_API`, but it didn't have any effect on the rest calls at all.

**To Reproduce**

1. Set Env Variables
2. Open site
3. Watch Dev tools for API calls
4. Observe that they don't go to the value that is set for `VUE_APP_URL_BASE_API`

**Expected behavior**
API calls (also SSE) should use `VUE_APP_URL_BASE_API`

**Environment (please complete the following information):**
 - OS: Kubernetes (K3s) running on Ubuntu 18.04
 - n8n Version 0.95.1
 - Using the docker Image

**Additional context**
I also set the following Environment Variables:

```
GENERIC_TIMEZONE: Europe/Berlin
N8N_BASIC_AUTH_ACTIVE: true
N8N_BASIC_AUTH_PASSWORD: xxxxx
N8N_BASIC_AUTH_USER: xxxxx
N8N_HOST: n8n.example.com
N8N_PORT: 5678
N8N_PROTOCOL: https
NODE_ENV: production
VUE_APP_URL_BASE_API: http://n8n.example.com:5678/
WEBHOOK_TUNNEL_URL: https://n8n.example.com/
```

**Comment by janober** [comment]
You are sadly right. It does not work as intended and does not seem to be technically possible to read that dynamically as n8n uses the prebuild files. Fixed the documentation:
https://github.com/n8n-io/n8n-docs/pull/319

So if you want it to work, you would have to build your own custom n8n image.

You can also find the full server setup guide with traefik (that it does not mess um SSE) here:
https://docs.n8n.io/reference/server-setup.html#server-setup

**Comment by Colton** [comment]
@janober Looking for a workaround for this issue right now. Currently spinning up N8N instances via PM2 Process manager. The N8N instances are accessible behind a reverse proxy (unique url for each instance).

Is there currently a working link for https://docs.n8n.io/reference/server-setup.html#server-setup ?

Any help would be greatly appreciated. Thank you!

CC // @andrewSeldenrust

**Comment by ffMathy** [comment]
Related: https://community.n8n.io/t/how-can-i-configure-vue-app-url-base-api-for-a-home-assistant-addon/81514

**Comment by ffMathy** [comment]
@janober I can confirm that setting `VUE_APP_URL_BASE_API` still doesn't do anything.

Is this a known issue? Can we re-open this one?

**Comment by tnde0tsprich** [comment]
This still does not work with env values from .env via docker compose.

**Comment by BhushanGit77** [comment]
@FlomoN @ffMathy @Colton @nsd0tsprich  I was also facing the similar issue. 
I wanted to access n8n via browser on url:  https://example.com/n8n. Setting VUE_APP_URL_BASE_API didn't work.

setup:
n8n container running locally on port 5678 behind reverse proxy. Reverse proxy accessible on port 443.

I tried following solution and it worked.
1) set env variable N8N_PATH= /n8n/
2) set env variable WEBHOOK_URL= https://example.com:443/n8n
3) set env variable N8N_PUSH_BACKEND=sse
4) On my reverse proxy side, I added logic to rewrite the path before sending to n8n, basically stripping away path prefix (ie. /n8n) so that requests at n8n arrive at n8n:5678 without path prefix.

eg. https://example.com:443/n8n/abc -> http://example.com:5678/abc

With this approach things started working for me. I am able to open n8n on my browser using custom url.
